mdoc/src/main/scala/mdoc/internal/markdown/Processor.scala

```diff
@@ -78,11 +78,12 @@ class Processor(implicit ctx: Context) {
   }
 
   def processPreInput(doc: MarkdownFile, custom: PreFenceInput): Unit = {
-    val PreFenceInput(block, input, Pre(mod, info)) = custom
+    val PreFenceInput(block: CodeFence, input, Pre(mod, info)) = custom
```

Comment:
Why the code annotation, can this throw?

Reply:
I think this is nothing more than a development artefact of me inserting a type to remind me explicitly what this was. I've removed it in a follow up commit - I don't think there's a functional reason for it - thanks - good spot.